.github/bin/languages/update.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+# Update the languages summary
+dotnet run -p .github/bin/languages
+
+# Format the documents
+npx prettier@2.0.4 --write languages/README.md languages/languages.json
+
+# Add the updated language summary files
+git add languages/README.md
+git add languages/languages.json
+
+# Check if there is nothing to commit (i.e. no changes to the languages summary)
+if [ -z "$(git status --porcelain)" ]; then
+    echo "No changes to the languages summary"
+    exit 0
+fi
+
+# Setup the git user (required to commit anything)
+git config --global user.email "github-actions[bot]@users.noreply.github.com"
+git config --global user.name "github-actions[bot]"
+
+echo "Committing updated languages summary"
+
+# Commit the changes
+git add .
+git commit -m "[CI] Update languages summary"
+git push
```

Comment:
Pushing straight to master will have the same issue that was reported on Slack earlier: https://github.com/exercism/v3/runs/798972403?check_suite_focus=true

A workaround would be pushing it to a branch and then having an auto merge account merge it (I think @iHiD mentioned he created a machine account for Exercism?)

Reply:
Indeed. I'll have to fix this